package.json

```diff
@@ -1,11 +1,10 @@
 {
   "name": "generator-patternpack",
-  "version": "0.9.1",
+  "version": "0.10.0",
   "description": "PatternPack yeoman generators to make creating a pattern library easy.",
   "dependencies": {
-    "lodash": "^3.10.1",
-    "patternpack": "^1.0.4",
```

Comment:
PatternPack was never used by the generators.  Removing this dependency really speeds things up